setup.py

```diff
@@ -37,6 +37,12 @@
 if os.path.exists('MANIFEST'):
     os.remove('MANIFEST')
 
+try:
+    import pypandoc
+    long_description = pypandoc.convert_file('README.md', 'rst')
+except (ImportError, IOError):
+    with open('README.md') as file:
```

Comment:
I can't reproduce the failure with the docs build on py2 or py3. I *think* you might be able to fix it by doing something like this though:

```
diff --git a/setup.py b/setup.py
index b3c97bd..7899df0 100644
--- a/setup.py
+++ b/setup.py
@@ -41,8 +41,8 @@ try:
     import pypandoc
     long_description = pypandoc.convert_file('README.md', 'rst')
 except (ImportError, IOError):
-    with open('README.md') as file:
-        long_description = file.read()
+    with open('README.md', 'rb') as file:
+        long_description = file.read().decode('utf8')

 if check_for_openmp() is True:
     omp_args = ['-fopenmp']
```